src/main/java/ua/com/juja/microservices/keepers/slackbot/model/dto/CustomRichMessage.java

```diff
@@ -0,0 +1,45 @@
+package ua.com.juja.microservices.keepers.slackbot.model.dto;
+
+import me.ramswaroop.jbot.core.slack.models.RichMessage;
+
+/**
+ * @author Dmitriy Lyashenko
+ */
+public class CustomRichMessage extends RichMessage{
```

Comment:
What is the advantage of CustomRichMessage against of RichMessage?

Reply:
I did it only for tests.